docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+# Benchmarks
+
+This document explains how to benchmark ASP.NET Core changes. It is intended for Microsoft employees.

```

Comment:
```suggestion
This document explains how to benchmark ASP.NET Core changes. Currently, only members of the .NET team at Microsoft have the permissions necessary to trigger performance runs on the official infrastructure.

```

Reply:
> Currently, only members of the .NET team at Microsoft

We usually just say "the .NET team".

Aside -- at some future point I'd be curious to understand how close we are to being able to allow other contributors to run these jobs. ie., whether there's anything needs to be more robust to make it merely a resource/cost issue that we could tackle by maintaining a trusted group. (We can discuss offline sometime)

Reply:
This is a hard requirement since jobs are triggered on dnceng/internal. Allowing any external user to run anything in this ring would compromise secrets. So we only accept users that have the merge permission on this repository as this would have the same effect. So I would believe that this will be possible when we allow external contributors to merge PRs to main or release/x.

Reply:
Gotcha